### PR TITLE
Transcribe DLA to Processing

### PR DESCRIPTION
**Pull request by FriskyDev**
This one was pretty simple and quick.

### CODE REVIEW

**Comment by FriskyDev** [comment]
image

File: CodingChallenges/CC_034_DLA/Processing/CC_034_DLA/CC_034_DLA.pde [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+// http://patreon.com/codingtrain
+// Code for this video: https://youtu.be/Cl_Gjj80gPE
+// Processing transcription: Chuck England
+
+import java.util.*;
+
+List<Walker> tree = new ArrayList<Walker>();
+List<Walker> walkers = new ArrayList<Walker>();
+//final float r = 4;
+final int maxWalkers = 50;
```
Comment (shiffman) [inline CodingChallenges/CC_034_DLA/Processing/CC_034_DLA/CC_034_DLA.pde]: `final` is not typically used in Processing examples. Though I'm ok with keeping it, it's a minor detail and perhaps nice for viewers to learn about!

File: CodingChallenges/CC_034_DLA/Processing/CC_034_DLA/Walker.pde [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  void show() {
+    noStroke();
+    if (stuck) {
+      fill(hu, 100, 100, 200);
+    } else {
+      fill(360, 0, 100);
+    }
+    ellipse(pos.x, pos.y, r * 2, r * 2);
+  }
+};
```
Comment (shiffman) [inline CodingChallenges/CC_034_DLA/Processing/CC_034_DLA/Walker.pde]:
```suggestion
}
```

File: CodingChallenges/CC_034_DLA/Processing/CC_034_DLA/CC_034_DLA.pde (hunk 1 above)
Comment (FriskyDev) [reply CodingChallenges/CC_034_DLA/Processing/CC_034_DLA/CC_034_DLA.pde]: Well, it can be confusing to new folks. I'll remove it.

**Pull request merged**